mino/minows/key/memory.go

```diff
@@ -0,0 +1,39 @@
+package key
+
+import (
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"go.dedis.ch/dela/core/store/kv"
+	"golang.org/x/xerrors"
+	"sync"
+)
+
+// cachedStorage offers a layer of in-memory caching on top of a diskStorage.
+type cachedStorage struct {
```

Comment:
This doesn't seem to be used in this PR, and it looks like premature optimization to me. Caching is hard, and should be avoided, except if it's in a hot-path, meaning it's called many thousand times a second.

KISS: Keep it Simple, Stupid

Reply:
removed